Src/Base/AMReX_BLProfiler.H

```diff
@@ -15,8 +15,6 @@
 #include <AMReX_Box.H>
 #endif
 
-#include <iostream>
-#include <ostream>
```

Comment:
you probably still need `ostream` here since it's appearing as an argument in the `operator<<`

Reply:
Yes, we need to add `isofwd` for this header to be independent.  But, because the `ostream` argument is a reference, not value, I think we are fine without `<iostream>`.

Reply:
Right, `isofwd` should work here. https://en.cppreference.com/w/cpp/header/iosfwd